docs/release/release-tasks.md

```diff
@@ -383,24 +383,26 @@ Information can be distributed via:
 
 * `sig-cluster-lifecycle` mailing list
   * Note: The person sending out the email should ensure that they are first part of the mailing list. If the email is sent out is not received by the community, reach out to the maintainers to unblock and approve the email.
-* Slack
+* #cluster-api Slack channel
 * Office hours
+* Release Team meetings
 * Cluster API book
-* ...
+* Conference talks
+* Github Issue (when communicating beta release to providers)
 
-Relevant information includes: (TBD)
+Relevant information includes:
 
-* Beta, RC, GA release
+* Beta, RC, GA and patch release
 * Start of code freeze
 * Implementation progress
-* ...
+* Release Team retrospective / pre-scheduled release cutting meetings
```

Comment:
I don't think this needs to be part of the communication of a release.

Reply:
Agreed, will remove it